regulations/static/regulations/js/source/views/main/search-results-view.js

```diff
@@ -78,30 +83,32 @@ var SearchResultsView = ChildView.extend({
     paginate: function(e) {
         e.preventDefault();
 
-        var page = $(e.target).hasClass('previous') ? this.page - 1 : this.page + 1,
-            config = {
-                query: this.query,
-                regVersion: this.resultsRegVersion,
-                page: page
-            };
+        var options = {
+          query: this.options.query,
+          docType: this.options.docType,
+          regVersion: this.options.regVersion,
```

Comment:
Yep, this is set in `MainView.createView` in an override specific to search results 😿. Also, this logic isn't changing here--just a minor reformat.